README.md

```diff
@@ -33,6 +33,170 @@ Users who join these GitHub teams agree to use the NASA Openscapes Hub only for
 
 Run large or parallel jobs over large geographic bounding boxes or over long temporal extents should be cleared with the NASA Openscapes Team by submitting an issue to this repo.   
 
+## Data Storage in the NASA Openscapes Hub
```

Comment:
Let's add this whole section to the end of the README, below Removal from Hub, so that people see that before getting deeper into storage and admin steps